### DISCUSSION TITLE
GitHub education pack

### DISCUSSION THREAD

**Discussion by angelarocha976** [community/community#148146] [GitHub Education]
### Hi everyone,

I have applied for education pack by approving myself as a student my application got approved and the benefits didn't recieved 
Kindly please process my educational benefits

**Comment by aayush-ojha** [comment]
Hi @angelarocha976,

The approval process usually gets completed under 48 hours, so please wait for some time and then check your email or the github education site to see if you have got the benefits or not.

**Comment by Olayiwolaaa** [comment]
If you got an approval mail, then yo have the benefit.
Check here: https://education.github.com/pack

**Comment by httperror505** [comment]
Github will send you an email indicating your application approval. From then on, you may access the perks such as CoPilot access in VS Code, Github Pro, and so much more.

**Comment by davevad93** [comment]
Hi @angelarocha976 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @angelarocha976 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
